app.py

```diff
@@ -1,3 +1,7 @@
+import os
+
 from drift.app import create_app
 
-create_app().run(host='0.0.0.0', port=8080)
+drift_debug = os.getenv('DRIFT_DEBUG')
+
+create_app().run(host='0.0.0.0', port=8080, debug=drift_debug)
```

Comment:
Is this a true/false value? You could answer my question by just adding this to the README :)

Reply:
updated!